### PR TITLE
Add pki nss-cert-find --cert option

### PR DESCRIPTION
**Pull request by edewata**
The `pki nss-cert-find` has been updated to provide an option to search for a cert in NSS database based on an existing cert file. This can be used to check whether the cert has been imported into the NSS database and to get the nickname and trust flags assigned to the cert.

### CODE REVIEW

**Review by ckelleyRH — APPROVED** [review]
LGTM!

File: base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertFindCLI.java [hunk 1]
```diff
@@ -29,15 +46,94 @@ public void printHelp() {
...
 
-        MainCLI mainCLI = (MainCLI) getRoot();
-        mainCLI.init();
+        option = new Option(null, "format", true, "Certificate format: PEM (default), DER");
+        option.setArgName("format");
+        options.addOption(option);
+    }
+
+    public Collection<X509Certificate> findCertByDERCert(byte[] derCert) throws Exception {
+
```
Comment (fmarco76) [inline base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertFindCLI.java:60]: Should never return more than one element so I would have preferred this to return an `X509Certificate`. The caller can add in the list.

**Review by fmarco76 — APPROVED** [review]
LGTM. Just a minor comment but feel free to merge as it is.

**Comment by edewata** [comment]
@ckelleyRH @fmarco76 I updated the code. Thanks!

**Pull request merged**